sites/public/pages/listings.tsx

```diff
@@ -175,11 +174,12 @@ const ListingsPage = () => {
             />
             <Select
               id="communityType"
-              name="communityType"
+              name={ListingFilterKeys.communityType}
               label={t("listingFilters.communityType")}
               register={register}
               controlClassName="control"
               options={communityTypeOptions}
+              defaultValue={filterState?.communityType}
```

Comment:
Above I mentioned changing the `communityType` filter to be just a `specialNeeds` filter. If we do that, then we need to translate between the drop down in this modal and the url format.

I think we have a few options:
- We can decouple the frontend and backend filter representations. So on the frontend we'd use a different filterState object that isn't `ListingFilterParams` and can be whatever shape it needs to be to match the form, and then in the encode/decode methods build the url with the filters.
- We can leave the `filterState` as an instance of `ListingFilterParams` and add some change handler to this Select that sets the right filter (senior housing or special needs) to true.

Reply:
Done. See comments above.